app/Http/Controllers/VisitPunchController.php

```diff
@@ -72,13 +72,7 @@ public function store(StoreVisitPunch $request)
                 $updatedSpaces = $this->updateUserSpacesFromSUMS($user);
             }
             if (null === $updatedSpaces) {
-                // Not a member of any SCC Billing Groups in SUMS
-                Log::info('Punch rejected - No default space(s) set for '.$gtid);
-
-                return response()->json([
-                    'status' => 'error',
-                    'error' => 'No default space(s) set for user.',
-                ], 422);
+                $updatedSpaces = collect([Space::where('id', $request->input('space_id'))->sole()]);
```

Comment:
I think this an equivalent but more concise version of this:
```suggestion
                $updatedSpaces = collect([Space:findOrFail($request->input('space_id'))]);
```